### PR TITLE
patch release v10.0.2

### PR DESCRIPTION
**Pull request by ank27**
### Summary of changes
patch release to fix billing issue

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 
+# 10.0.2 December 13, 2021
+
+## Bug fixes 🐞
+* Fix billing issue when upgrading Mapbox Maps SDK from v9 to v10.
+
+## Dependencies
+* Bump common to v20.0.3.
+
 # 10.0.1 November 26, 2021
```
Comment (kiryldz) [inline CHANGELOG.md:13]: we want to leave 10.0.1 changelog here, correct?

**Review by kiryldz — APPROVED** [review]
LGTM with 1 question

File: CHANGELOG.md (hunk 1 above)
Comment (ank27) [reply CHANGELOG.md:13]: yeah, we will keep it as a notification on why the older patch doesn't work as expected. cc @pengdev

Comment (kiryldz) [reply CHANGELOG.md:13]: do we need to update changelog entry for 10.0.1 then? cause right now we have 2 similar patches per changelog while only latest one will be available as a binary

**Review by yunikkk — APPROVED** [review]

**Review by pengdev — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.2 December 13, 2021
+
+## Bug fixes 🐞
+* Fix billing issue when upgrading Mapbox Maps SDK from v9 to v10.
+
+## Dependencies
+* Bump common to v20.0.3.
```
Comment (pengdev) [inline CHANGELOG.md:11]: nit: link to this PR

File: CHANGELOG.md (hunk 1 above)
Comment (ank27) [reply CHANGELOG.md:13]: I don't think we need to change the entry for 10.0.1 as it was mentioned that `a new patch will be issued shortly` in the changelog. so this one will work as expected.

**Pull request merged**